eng/scripts/Collect-Changelogs.ps1

```diff
@@ -37,5 +42,4 @@ Get-ChildItem "$repoRoot/sdk" -Filter CHANGELOG.md -Recurse | Sort-Object -Prope
     }
 }
 
-$InstallNotes | Write-Host;
-$ReleaseNotes | Write-Host;
\ No newline at end of file
+return $InstallNotes, $ReleaseNotes
```

Comment:
Tip: the return statement is unnecessary, but doesn't hurt. It's effectively the same as:

```powershell
$InstallNotes, $ReleaseNotes
return
```

Great for it you want to return values and then exit, but since you're at the end.

It's fine to keep it. Just a tip. ;)